jenkins-prometheus-exporter.py

```diff
@@ -279,12 +293,23 @@ def scrape():
         jenkins_builds_total_family.add_metric(labels, value)
 
     jenkins_build_errors_total_family = CounterMetricFamily(
-        'jenkins_build_errors_total', 'Count of all jenkins build errors', labels=BUILD_LABELS
+        'jenkins_build_errors_total',
+        'Count of all jenkins build errors',
+        labels=BUILD_LABELS,
     )
     error_builds = only(builds, states=error_states)
     for value, labels in jenkins_builds_total(jobs, error_builds):
         jenkins_build_errors_total_family.add_metric(labels, value)
 
+    jenkins_build_unstable_total_family = CounterMetricFamily(
+        'jenkins_build_unstable_total',
+        'Count of all jenkins build unstable',
```

Comment:
This is the human readable help text which will probably save us some day. Can you rephrase as `Count of all jenkins builds that resulted in the unstable state` or something like that?

Reply:
Done.  "Count of all jenkins builds in the unstable state"